.github/actions/pkg-build/action.yml

```diff
@@ -27,7 +27,7 @@ runs:
       shell: bash
 
     - name: Create package
-      run: python setup.py sdist bdist_wheel
+      run: python -m build
```

Comment:
Wbat is the advantage of -m build? The previous way is the standard and doesnt require an extra dependency

Reply:
`python -m build`:

- This command is part of the newer PEP 517 and PEP 518 packaging standards and is associated with the `pyproject.toml` file.
- It uses the "build backend" specified in the `pyproject.toml` file to build distribution packages.
- It is part of the modern packaging ecosystem and is recommended for new projects.
The build command is more declarative and less reliant on complex setup.py scripts.

Recommendation:

If you're starting a new Python project or working on a project that uses modern packaging standards, it's recommended to use `python -m build` along with a pyproject.toml file. This approach is more modern, standardized, and better suited for Python packaging best practices.

However, if you're working with an older project or a project that relies on `setuptools` and has a setup.py script, you should continue using `python setup.py sdist` to build your distribution package.

In summary, you can see it as a continuous dependency update...

Reply:
> However, if you're working with an older project or a project that relies on setuptools and has a setup.py script, you should continue using python setup.py sdist to build your distribution package.

Don't we fall into this category? Our packaging logic is built with `setuptools` and we have a `setup.py` file

Reply:
yes, but if the official recommendation is to stick to it, I agree with @carmocca to do so (especially since our `setup.py` is relatively complex)